tools/PrebidValidator/Dr.Prebid/TestSummary/SDKIntegrationValidation/PBVPrebidSDKValidator.m

```diff
@@ -77,7 +77,7 @@ - (BOOL)setupPrebidAndRegisterAdUnits {
     @try {
         // Prebid Mobile setup!
         [self setupPrebidLocationManager];
-        [self setPrebidTargetingParams];
+        //[self setPrebidTargetingParams];
```

Comment:
can we just remove this line and this method?

Reply:
I just wanted to keep this method in case in future you want to targeting params we can just use this method